client/reader/following/intro.jsx

```diff
@@ -1,107 +0,0 @@
-import { Gridicon } from '@automattic/components';
-import { localize } from 'i18n-calypso';
-import { Component } from 'react';
-import { connect } from 'react-redux';
-import readerBackground from 'calypso/assets/images/reader/reader-intro-background.svg';
-import readerImage from 'calypso/assets/images/reader/reader-intro-character.svg';
-import QueryPreferences from 'calypso/components/data/query-preferences';
-import cssSafeUrl from 'calypso/lib/css-safe-url';
-import { isUserNewerThan, WEEK_IN_MILLISECONDS } from 'calypso/state/guided-tours/contexts';
-import { savePreference } from 'calypso/state/preferences/actions';
-import { getPreference } from 'calypso/state/preferences/selectors';
-import { recordReaderTracksEvent } from 'calypso/state/reader/analytics/actions';
-
-class FollowingIntro extends Component {
-	componentDidMount() {
-		this.recordRenderTrack();
-	}
-
-	componentDidUpdate( prevProps ) {
-		if ( this.props.isNewReader !== prevProps.isNewReader ) {
-			this.recordRenderTrack();
-		}
-	}
-
-	dismiss = () => {
-		this.props.recordReaderTracksEvent( 'calypso_reader_following_intro_dismiss' );
-		return this.props.savePreference( 'is_new_reader', false );
-	};
-
-	handleManageLinkClick = () => {
-		this.props.recordReaderTracksEvent( 'calypso_reader_following_intro_link_clicked' );
-		return this.props.savePreference( 'is_new_reader', false );
-	};
-
-	recordRenderTrack = ( props = this.props ) => {
-		if ( props.isNewReader === true ) {
-			this.props.recordReaderTracksEvent( 'calypso_reader_following_intro_render' );
-		}
-	};
-
-	render() {
-		const { isNewReader, isNewUser, translate } = this.props;
-
-		if ( ! isNewReader || ! isNewUser ) {
-			return null;
-		}
-
-		const linkElement = <a onClick={ this.handleManageLinkClick } href="/read/subscriptions" />;
-
-		return (
-			<header
-				className="following__intro"
```

Comment:
This classname is unused, but there are some styles that correspond to it. Let's clean them up:

https://github.com/Automattic/wp-calypso/blob/55ba5544f93388dcb81ee385d71dc8668b40a774/client/reader/following/style.scss#L45-L172